src/spryker/helm/preview/Chart.yaml

```diff
@@ -1,6 +1,7 @@
 apiVersion: v1
-name: app-preview
+name: preview
 description: Base helm chart for the app-preview environment
 version: 0.0.1
+apiVersion: v1
```

Comment:
duplicated now

Reply:
ach, sorry! I resolved the conflict via the web browser... I will fix it from PhpStorm 😂